custom/export_xml.php

```diff
@@ -252,25 +255,36 @@ function addInsurance($row, $seq) {
 ?>
 <html>
 <head>
-<?php html_header_show();?>
-<link rel=stylesheet href="<?php echo $css_header;?>" type="text/css">
-<title><?php xl('Export Patient Demographics','e'); ?></title>
+<?php Header::setupHeader(); ?>
+<title><?php echo xlt('Export Patient Demographics'); ?></title>
 </head>
 <body>
-
-<p><?php xl('The exported data appears in the text area below.  You can copy and
-paste this into an email or to any other desired destination.','e'); ?></p>
-
-<center>
 <form>
```

Comment:
I know this form tag was here before your changes, but we can go ahead and pull it, there's not need for it